documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc

```diff
@@ -15,6 +15,7 @@ Example dashboards are provided in the `examples/metrics` directory as JSON file
 * `strimzi-kafka-mirror-maker-2.json`
 * `strimzi-operators.json`
 * `strimzi-kafka-bridge.json`
+* `strimzi-cruise-control.json`
 
 The example dashboards are a good starting point for monitoring key metrics, but they do not represent all available metrics.
 You can modify the example dashboards or add other metrics, depending on your infrastructure.
```

Comment:
At the end of this module there are bullet lists showing the metrics on each dashboard.

I suggest adding a new metrics list for Cruise Control.

Reply:
Actually, that information is in the new assembly. I'm not sure you need to repeat it again.

Reply:
Yeah the list is already in the new assembly as for the bridge, I don't think we should repeat it again.